src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -65,16 +70,21 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 				await addDAO(gnosisAddress)
 				await editDAO({gnosisAddress, name: daoName})
 				toastSuccess("DAO successfully created!")
+				setCreated("create-success")
 				afterCreate()
 			} catch (e) {
 				console.error(e)
+				setCreated("error")
 				toastError("Failed to create DAO")
 			}
 			setProcessing(false)
+		} else if (stage === "load-existing") {
+			console.log("TODO")
 		}
 	}
 
 	const submitButtonDisabled =
+		typeof newGnosis === "undefined" ||
```

Comment:
If `newGnosis` could be false - it should be rather in form of `isNewGnosis`. `newGnosis` sounds more like "newly created/added/whatever gnosis"